### DISCUSSION TITLE
You are using Node.js 18.7.0. For Next.js, Node.js version >= v18.17.0 is required.

### DISCUSSION THREAD

**Discussion by Samsursekh** [vercel/next.js#61674] [Help]
### Summary

Although I am using node version  `20.11.0.`  Still, I am getting the alert  **You are using Node.js 18.7.0. For Next.js, Node.js version >= v18.17.0 is required.**

and I am not using this version `18.7.0` anywhere in the code.   I have deleted my npm cache deleted Nodejs and re-installed it so that it works well.  it was fine for the react application. but when I try to use Nextjs after entering `npm run dev`, it occurring the issues.

Also, I have changed the node version with `nvm` . still not working. Please help me.  

ASUS@LAPTOP-0EOJDOHN MINGW64 ~/OneDrive/Desktop/prac-app-next (master)
$ npm -v
10.2.4

ASUS@LAPTOP-0EOJDOHN MINGW64 ~/OneDrive/Desktop/prac-app-next (master)
$ nvm -v
0.39.0

ASUS@LAPTOP-0EOJDOHN MINGW64 ~/OneDrive/Desktop/prac-app-next (master)
$ node -v
v20.11.0

ASUS@LAPTOP-0EOJDOHN MINGW64 ~/OneDrive/Desktop/prac-app-next (master)
$ npm run dev

> prac-app-next@0.1.0 dev
> next dev

`You are using Node.js 18.7.0. For Next.js, Node.js version >= v18.17.0 is required.`

After making changes I restarted the terminal and also restarted my computer as well.

![nextjs-problem](https://github.com/vercel/next.js/assets/101379073/7e5db353-5256-4e3f-9947-f01e7ee76631)

![next-js-issu-2](https://github.com/vercel/next.js/assets/101379073/c6853199-8815-47ff-8b52-08250a73047a)

**Comment by icyJoseph** [comment]
Hi,

Could you run `node` and then type in `process.versions.node`, and see what that prints?

```shell
node
Welcome to Node.js v18.18.0.
Type ".help" for more information.
> process.versions.node
'18.18.0'
> .exit
```

I'll take a look at how Next.js calculates the runtime version used though. If you are using a Node Version Manager, there can be some issues...

**Reply by Kampouse** [reply]
https://github.com/vercel/next.js/blob/5717b9a14de7c97c16b2920049d2c3f0ce1ec849/packages/next/src/cli/next-info.ts#L78
this  is the code your looking for
i was my thing with pnpm and it bundle its own nodejs leaving me with an older version of nodejs

**Comment by l3hash** [comment]
Hey, have you been able to find a solution to that problem?

**Reply by shaeebali** [reply]
I have had a long winded research campaign regarding repeated issues with nextjs and the node error messages. Run the following command in the project directory, then try npm run dev and it should work:

npm i next@13 react@latest react-dom@latest eslint-config-next@13

**Comment by shaeebali** [comment]
I have had a long winded research campaign regarding repeated issues with nextjs and the node error messages. Run the following command in the project directory, then try npm run dev and it should work:

npm i next@13 react@latest react-dom@latest eslint-config-next@13
